### PR TITLE
Feature/newtickettypes

### PR DESCRIPTION
**Pull request by xx4673**
tciket_type1.0 & ticket1.0 & comments1.0

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -60,6 +59,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'foreman', '~> 0.87.2'
```
Comment (annielin28815) [inline Gemfile:62]: 是否刪除此行？

**Review by samliuforwork — APPROVED** [review]
checked

**Review by kihifung — APPROVED** [review]
checked

**Review by annielin28815 — APPROVED** [review]
checked!

**Pull request merged**